EpsilonAuxBmsV2/Tests/tests/OrionInterfaceTest/OrionInterfaceTest.c

```diff
@@ -17,241 +12,75 @@ osThreadId_t commonContactorGatekeeperTaskHandle;
-
-
-    setNominalAuxStatus();
-    AuxStatus expectedAuxStatus = auxStatus;
-    expectedAuxStatus.allowCharge = 0;
-    expectedAuxStatus.allowDischarge = 0;
-    expectedAuxStatus.strobeBmsLight = 1;
-    expectedAuxStatus.dischargeShouldTrip = 1;
-    expectedAuxStatus.chargeShouldTrip = 1;
-
+// void test_onlyUpdateAllowCharge() {
+//     AuxStatus auxStatusToUpdate;
+//     auxStatusToUpdate.allowCharge = 1;
+//     message.highCellVoltage = 42000;
+//     updateAllowChargeAndAllowDischarge(&message, &auxStatusToUpdate);
+//     TEST_ASSERT_EQUAL_MESSAGE(0, auxStatusToUpdate.allowCharge, "auxStatusToUpdate.allowCharge failed to update");
+// }
+
+// void test_onlyUpdateAllowDischarge() {
+//     AuxStatus auxStatusToUpdate;
+//     auxStatusToUpdate.allowDischarge = 1;
+//     message.lowCellVoltage = 25000;
+//     updateAllowChargeAndAllowDischarge(&message, &auxStatusToUpdate);
+//     TEST_ASSERT_EQUAL_MESSAGE(0, auxStatusToUpdate.allowDischarge, "auxStatusToUpdate.allowDischarge failed to update");
+// }
+
+// TODO: Find a better naming convetion for unit tests
+void test_osMessageQueueGetTimeout() {
+    AuxStatus expectedAuxStatus = {.orionCanReceivedRecently = 0};
+    auxBmsContactorState.startupDone = 0;
+    uint32_t contactorControlEventFlags = 0;
+    
+    osMessageQueueGet_ExpectAndReturn(orionInterfaceQueue, &message, NULL, ORION_QUEUE_TIMEOUT, osErrorTimeout);
+    HAL_GPIO_ReadPin_ExpectAndReturn(ORION_DISCHARGE_ENABLE_SENSE_GPIO_Port, ORION_DISCHARGE_ENABLE_SENSE_Pin, 1);
+    HAL_GPIO_ReadPin_ExpectAndReturn(ORION_CHARGE_ENABLE_SENSE_GPIO_Port, ORION_CHARGE_ENABLE_SENSE_Pin, 1);
+    osMutexAcquire_ExpectAndReturn(auxTripMutex, MUTEX_TIMEOUT, osError);
+    osMutexAcquire_ExpectAndReturn(auxStatusOrionInterfaceMutex, MUTEX_TIMEOUT, osOK);
+    osMutexRelease_ExpectAndReturn(auxStatusOrionInterfaceMutex, 0);
+    osEventFlagsSet_ExpectAndReturn(contactorControlEventBits, contactorControlEventFlags, 0);
+    
     orionInterface(&message);
-
-    assertAuxStatusEqual(expectedAuxStatus);
+    TEST_ASSERT_EQUAL_MESSAGE(expectedAuxStatus.orionCanReceivedRecently, auxStatus.orionCanReceivedRecently, "auxStatus.orionCanReceivedRecently failed to update");
 }
 
-// Orion reports a cell voltage above the internal Aux BMS range
-// Test that the charge contactor off event is triggered & statuses are correct
-void test_TooHighMaxCellVoltageShouldTurnOffChargeAndUpdateStatuses()
-{
-    setOrionInterfaceOsExpects(1);
-    setTripExpects(0, 0, 0);
-    setContactorState(CLOSED, CLOSED, 1);
-    setContactorEventFlagsExpects(CHARGE_OFF);
+void test_shouldDisconnectContactors() {
+    AuxStatus expectedAuxStatus = {
+        .orionCanReceivedRecently = 0,
+        .allowCharge = 0,
+        .allowDischarge = 0,
+        .strobeBmsLight = 1
+    };
+    AuxBmsContactorState expectedAuxBmsContactorState = {.contactorsDisconnected = 1};
+    uint32_t contactorControlEventFlags = 0x28; // CHARGE_OPENED and DISCHARGE_OPENED
```

Comment:
Change these to use the actual flag defines (CHARGE_OPENED | DISCHARGE_OPENED)

Reply:
Also, this variable isn't being used.

Reply:
Woops, I forgot to commit the latest changes.